app/courses/[courseId]/page.module.css

```
@@ -51,172 +51,175 @@
+  align-items: baseline;
+  justify-content: space-between;
+  gap: 1rem;
+  padding-bottom: 0.55rem;
+  border-bottom: 1px solid #e3e6ea;
 }
 
-.courseHeading {
-  margin: 0.9rem 0 2.4rem;
+.infoTitle {
+  margin: 0;
   font-family: 'Lato', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)